site/assets/scss/_sidebar.scss

```diff
@@ -44,19 +40,18 @@
     content: escape-svg($sidebar-collapse-icon);
   }
 
-  &.active {
-    .bd-sidenav-group-link::before {
-      transform: rotate(90deg);
-    }
+  [aria-expanded="true"] {
```

Comment:
Not a fan of using attributes to style things. We should be able to style things based on the `.collapsed` class. The class should be added to the collapsed items by default though.

Reply:
In fact it's lighter using attribute selector: the equivalent selector would be something like `&.has-children .bd-sidenav-group-link:not(.collapsed)`.

Would really prefer the latter?

Question, too: aren't `.bd-sidenav-group` and `.has-children` redundant? I can't find a case where one of those would be set without the other…